config.json

```diff
@@ -302,7 +302,1261 @@
       "discover",
       "mastercard",
       "visa"
-    ]
+    ],
```

Comment:
Hey @junedkazi , as we decided to create the input fields manually for the card management feature, I created three new attributes on the config.json to add the default countries, states and form validations. I would like to know if this the right place for storing those values? Thanks

Reply:
why do we want to hard code the counties and states list ? Can't we make a call to bcapp and fetch the information ?

Reply:
I agree, I don't think we should be hardcoding this list. //cc @precariouspanther @zackangelo @chrisboulton another possibly opportunity to use the new service that we have been discussing.

Reply:
Yes please, we've had several occasions where we have had to update these lists - it'd be preferable to avoid fragmenting this. The other concern is these values need to be matched to names within our lists which change (rather than the ISO codes) so there is a risk that changes to the official list would be a backwards breaking change for this that we wouldn't realise.
Please consider driving this from either @aleachjr 's geo service or BCApp's current definition (Which will be migrated)